build_and_check_linux_wheels.sh

```diff
@@ -14,11 +14,6 @@ do
 
     PYTHON=${PYVERSION:2:1}.${PYVERSION:3:1}
 
-    # Remove this when python 3.7 is released
```

Comment:
I think `install_osx.sh` need a similar modification. Could you have a quick look?